With the dependency missing from type checking, we're getting some failures:

```
nox > mypy bigframes tests/system tests/unit --check-untyped-defs --explicit-package-bases '--exclude="^third_party"'
tests/system/large/streaming/test_pubsub.py:24: error: Module "google.cloud" has no attribute "pubsub"  [attr-defined]
Found 1 error in 1 file (checked 380 source files)
nox > Command mypy bigframes tests/system tests/unit --check-untyped-defs --explicit-package-bases '--exclude="^third_party"' failed with exit code 1
nox > Session mypy failed.
Error: Process completed with e
```

--- a/tests/system/large/streaming/test_pubsub.py
+++ b/tests/system/large/streaming/test_pubsub.py
@@ -21,7 +21,7 @@ import pytest
 import bigframes
 
 pytest.importorskip("google.cloud.pubsub")
-from google.cloud import pubsub  # noqa
+from google.cloud import pubsub  # type: ignore # noqa
 
 
 def resource_name_full(project_id: str, resource_type: str, resource_id: str):

